polkadot-parachains/integritee-runtime/src/lib.rs

```diff
@@ -550,6 +552,140 @@ impl pallet_treasury::Config for Runtime {
+	type MotionDuration = TechnicalMotionDuration;
+	/// The maximum number of Proposlas that can be open in the technical committee at once.
+	type MaxProposals = TechnicalMaxProposals;
+	/// The maximum number of technical committee members.
+	type MaxMembers = TechnicalMaxMembers;
+	type DefaultVote = pallet_collective::MoreThanMajorityThenPrimeDefaultVote;
+	type WeightInfo = pallet_collective::weights::SubstrateWeight<Runtime>;
+}
+
+pub type EnsureRootOrTwoThirdsTechnicalCommittee = EnsureOneOf<
+	EnsureRoot<AccountId>,
+	pallet_collective::EnsureProportionAtLeast<AccountId, TechnicalCommitteeInstance, 2, 3>,
+>;
+
+pub type EnsureRootOrAllTechnicalCommittee = EnsureOneOf<
+	EnsureRoot<AccountId>,
+	pallet_collective::EnsureProportionAtLeast<AccountId, TechnicalCommitteeInstance, 1, 1>,
+>;
+
+parameter_types! {
+	pub const LaunchPeriod: BlockNumber = prod_or_fast!(5 * DAYS, 1 * MINUTES);
+	pub const VotingPeriod: BlockNumber = prod_or_fast!(5 * DAYS, 1 * MINUTES);
+	pub FastTrackVotingPeriod: BlockNumber = prod_or_fast!(3 * HOURS, 1 * MINUTES);
+	pub const MinimumDeposit: Balance = 100 * TEER;
+	pub EnactmentPeriod: BlockNumber = prod_or_fast!(2 * DAYS, 1 * MINUTES);
+	pub const CooloffPeriod: BlockNumber = prod_or_fast!(7 * DAYS, 1 * MINUTES);
+	pub const InstantAllowed: bool = true;
+	pub const MaxVotes: u32 = 100;
+	pub const MaxProposals: u32 = 100;
+	pub const PreimageByteDeposit: Balance = 10 * MICROTEER; // 10 times a tx fee
+}
+
+impl pallet_democracy::Config for Runtime {
+	type Proposal = Call;
+	type Event = Event;
+	type Currency = Balances;
+	type EnactmentPeriod = EnactmentPeriod;
+	type LaunchPeriod = LaunchPeriod;
+	type VotingPeriod = VotingPeriod;
+	type VoteLockingPeriod = EnactmentPeriod;
+	type MinimumDeposit = MinimumDeposit;
+	//// Only root is able to schedule a proposal to become a SuperMajorityApprove (default decline)
+	/// referendum once it is is legal for an externally proposed referendum
+	type ExternalOrigin = EnsureRootOrHalfCouncil;
+	/// Everyone is able to schedules a proposal to become a majority-carries (Simple Majority)
+	/// referendum once it is legal for an externally proposed referendum
+	type ExternalMajorityOrigin = EnsureRootOrHalfCouncil;
+	/// Only root is able to schedule a proposal to become a SuperMajorityAgainst (default accept)
+	/// referendum once it is legal for an externally proposed referendum
+	type ExternalDefaultOrigin = EnsureRootOrAllCouncil;
+	/// Two thirds of the technical committee can have an ExternalMajority/ExternalDefault vote
+	/// be tabled immediately and with a shorter voting/enactment period.
+	type FastTrackOrigin = EnsureRootOrTwoThirdsTechnicalCommittee;
+	type InstantOrigin = EnsureRootOrAllTechnicalCommittee;
+	type InstantAllowed = InstantAllowed;
+	type FastTrackVotingPeriod = FastTrackVotingPeriod;
+	// To cancel a proposal which has been passed.
+	type CancellationOrigin = EnsureRootOrTwoThirdsCouncil;
+	type BlacklistOrigin = EnsureRoot<AccountId>;
```

Comment:
This should probably be changed because we will remove sudo soon?

Reply:
This is actually the same as in Kusama: https://github.com/paritytech/polkadot/blob/master/runtime/kusama/src/lib.rs#L646

Root can still be called, but only via referendum. Should the Council be able to blacklist a proposal?

Reply:
I think yes.

Reply:
To what rate?

Reply:
I'd say `AllCouncil` this is the closest to root that kusama uses.